src/crate_owner_invitation.rs

```diff
@@ -112,11 +112,7 @@ pub fn handle_invite(req: &mut Request) -> CargoResult<Response> {
     if crate_invite.accepted {
         accept_invite(req, conn, crate_invite)
     } else {
-        #[derive(Serialize)]
-        struct R {
-            crate_owner_invitation: InvitationResponse,
-        }
-        Ok(req.json(&R { crate_owner_invitation: crate_invite }))
+        decline_invite(req, conn, crate_invite)
```

Comment:
I really like how this ended up, the code's easier to read and well organized! 💯